metadata/2024-07-17-enable-bn254/enable_bn254_algebra.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable BN254 algebra",
+  "description": "Enable BN254 algebra",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
+  "discussion_url": "https://github.com/aptos-labs/aptos-core"
```

Comment:
Put the real url to the AIP discussion instead

Reply:
fixed